test/clj/game_test/cards/assets.clj

```diff
@@ -1696,7 +1696,7 @@
         (core/rez state :corp (refresh gs))
         (card-ability state :corp (refresh gs) 0)
         (is (refresh gs) "Gene Splicer is still in remote")
-        (is (zero? (count (get-scored state :corp))) "Score area is still empty")))))
+        (is (empty (get-scored state :corp)) "Score area is still empty")))))
```

Comment:
Watch out. `(empty coll)` returns an empty collection of the same type. You're looking for `(empty? coll)` :wink: